canard/functions.php

```diff
@@ -306,6 +306,7 @@ function canard_scripts() {
  */
 function canard_editor_styles() {
 	wp_enqueue_style( 'canard-editor-block-style', get_template_directory_uri() . '/editor-blocks.css' );
+	wp_enqueue_style( 'canard-block-style', get_template_directory_uri() . '/blocks.css' );
```

Comment:
I think its probably better to enqueue this before the editor styles so that if they ever target the same selectors the editor styles win.